datapackage_pipelines/manager/tasks.py

```diff
@@ -10,11 +10,17 @@
 
 from .runners import runner_config
 
+SINK = os.path.join(os.path.dirname(__file__),
+                    '..', 'lib', 'internal', 'sink.py')
 
 async def enqueue_errors(step, process, queue):
     out = process.stderr
     while True:
-        line = await out.readline()
+        try:
+            line = await out.readline()
+        except ValueError:
+            logging.error('Received a too long log line (>64KB), truncated')
```

Comment:
No, it's being discarded altogether and the error log is shown instead.

I think that log lines should be used for human readable text - limiting log lines at 64K is quite reasonable, no?